core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/KeyAlreadyExistsException.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This class is not needed, please remove and use instead `org.apache.syncope.core.persistence.api.dao.DuplicateException` (see above).